### PR TITLE
Homework8: Ekaterina Zavadskaya

### PR DESCRIPTION
**Pull request by KateZavadskaya**
Feat(hm8): add homework8

Docs: hm8_1.py - hm8_3.py

### CODE REVIEW

**Comment by KateZavadskaya** [comment]
@Stml89 , success with homework8 💃, please

File: hm8_1.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+            count_j = count_j + 1
+    return (count_i <= 1) and (count_j <= 1)
+
+
+print(f"1 {solution([1, 3, 2])}")
+print(f"2 {solution([1, 3, 2, 1])}")
+print(f"3 {solution([1, 2, 1, 2])}")
+print(f"4 {solution([1, 3, 2, 1])}")
+print(f"5 {solution([1, 2, 3, 4, 5, 3, 5, 6])}")
+print(f"6 {solution([40, 50, 60, 10, 20, 30])}")
```
Comment (Stml89) [inline hm8_1.py]: better to use `assert`, instead of print() to check that it work as expected

File: hm8_3.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+                    sum_odd += digit
+                else:
+                    doubled = digit * 2
+                    if doubled > 9:
+                        sum_even += doubled - 9
+                    else:
+                        sum_even += doubled
+            return (sum_even + sum_odd) % 10 == 0
+        break
+    return None
```
Comment (Stml89) [inline hm8_3.py]: return False

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: hm8_1.py (hunk 1 above)
Comment (KateZavadskaya) [reply hm8_1.py]: done 👍, also according to this comment add changes to hm8_2.py, please

File: hm8_3.py (hunk 2 above)
Comment (KateZavadskaya) [reply hm8_3.py]: done 👍, also according to this comment add changes to hm8_2.py, please

File: hm8_1.py (hunk 1 above)
Comment (Stml89) [reply hm8_1.py]: Nice, that you have checked another file and corrected issues there as well! Well done!

Comment (Stml89) [reply hm8_1.py]: Also, it is better if you have more tests to your code:
```
assert solution([1, 3, 2]) == ..., ...
...
assert solution([1, 2, 3, 4, 5, 3, 5, 6]) == ..., ...
...
assert solution([40, 50, 60, 10, 20, 30] == ..., ...
```

**Review by Stml89 — APPROVED** [review]

File: hm8_1.py (hunk 1 above)
Comment (KateZavadskaya) [reply hm8_1.py]: I will take it into account in the next homework, thank you!🌸

**Pull request merged**